evaluation/longbench/calculate_metrics.py

```diff
@@ -0,0 +1,221 @@
+import re
+import string
+
+import jieba
+from fuzzywuzzy import fuzz
+import difflib
+
+from typing import List
+from collections import Counter
+from rouge import Rouge
+
+
+import os
+import json
+import argparse
+import numpy as np
+import logging
+from tqdm import tqdm
+
+
+def calculate_metrics(df):
+    preds = df["predicted_answer"].tolist()
+    labels = df["answer"].tolist()
+    task = df["task"].tolist()[0]
+    all_classes = df["all_classes"].tolist()[0]
+    return scorer(task, preds, labels, all_classes)
```

Comment:
Could you homogenize function arguments such that it reads
`return scorer(dataset, predictions, answers, all_classes)` (see function names in scorer)?